### PR TITLE
Initialize node-esclient

### PR DESCRIPTION
**Pull request by JrSchild**
### Context
Create a utility package for our elasticsearch client

### What has been done
- test cases
- npm scripts
- test coverage
- Travis integration
- README with buttons

### How to test
- Run `npm test`
- Test coverage at 100%

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ module.exports = function ESClient(config) {
     this.error = log.error;
     this.warning = log.warn;
 
-    // this can not be an arrow function because we require access to arguments.
+    // this can not be an arrow function because we require access the arguments.
```
Comment (edwardsmit) [inline index.js]: That's not a correct sentence

File: README.md [hunk 2]
```diff
@@ -1 +1,24 @@
...
+
+[![build:?](https://img.shields.io/travis/Enrise/node-esclient.svg?style=flat-square)](https://travis-ci.org/Enrise/node-esclient)
+[![Coverage Status](https://img.shields.io/coveralls/Enrise/node-esclient/master.svg?style=flat-square)](https://coveralls.io/github/Enrise/node-esclient?branch=master)
+[![dependencies:?](https://img.shields.io/david/Enrise/node-esclient.svg?style=flat-square)](https://david-dm.org/Enrise/node-esclient)
+[![devDependencies:?](https://img.shields.io/david/dev/Enrise/node-esclient.svg?style=flat-square)](https://david-dm.org/Enrise/node-esclient)
+
+> A simple wrapper around the [elasticsearch](https://github.com/elastic/elasticsearch-js) client with integrated [enrise-logger](https://github.com/Enrise/node-logger).
+
+### Installation
+NPM: `npm install enrise-esclient enrise-logger --save`
```
Comment (JrSchild) [inline README.md]: Yes we do because the logger needs to be intialized first.

File: index.js (hunk 1 above)
Comment (JrSchild) [inline index.js]: I need #c0ff33 ☕️

File: README.md (hunk 2 above)
Comment (edwardsmit) [inline README.md]: It is only a necessity if you want to configure the logger, right? And I would prefer that situation to be handled via the configuration of the ESClient. I find it awkward to have to have a separate package required and configured for this package to work.

**Comment by JrSchild** [comment]
🛎

**Pull request merged**